src/Temporalio/Bridge/OptionsExtensions.cs

```diff
@@ -131,7 +131,8 @@ internal static class OptionsExtensions
             return new Interop.LoggingOptions()
             {
                 filter = scope.ByteArray(options.Filter.FilterString),
-                forward = (byte)0,
+                // Forward callback is set in the Runtime constructor
+                // forward_to = ???
```

Comment:
Leftover line?

Reply:
Intentional, as a way to say "I know I am not setting this field here"